ApiBundle/Tests/Transformer/NodeGroupRoleTransformerTest.php

```diff
@@ -82,8 +91,8 @@ public function testTransform($nodeId, $role, $accessType)
     public function provideTransformData()
     {
         return array(
-            array('foo', 'bar', NodeGroupRoleInterface::ACCESS_GRANTED),
-            array('bar', 'foo', NodeGroupRoleInterface::ACCESS_DENIED),
+            array('foo', 'bar', NodeGroupRoleInterface::ACCESS_GRANTED, true),
+            array('bar', 'foo', NodeGroupRoleInterface::ACCESS_DENIED, false),
```

Comment:
among the methods using this provider, testTransform ( https://github.com/open-orchestra/open-orchestra-cms-bundle/blob/update_inherit_node_group_role/ApiBundle/Tests/Transformer/NodeGroupRoleTransformerTest.php#L73 ) accepts only 3 arguments